common/src/main/java/com/wynntils/models/statuseffects/type/StatusEffect.java

```diff
@@ -84,4 +105,23 @@ public int compareTo(StatusEffect effect) {
                 .compare(this.getModifier().getString(), effect.getModifier().getString())
                 .result();
     }
+
+    private Pair<StyledText, Optional<Double>> readModifierValueAndSuffix(StyledText modifier) {
+        String modifierStr = modifier.getStringWithoutFormatting();
+        if (modifierStr.isEmpty()) {
+            return new Pair<>(StyledText.EMPTY, Optional.empty());
+        }
+        int start = Math.max(modifierStr.indexOf('+') + 1, 0);
+
+        int end = Math.max(modifierStr.indexOf('%'), modifierStr.indexOf('/'));
```

Comment:
I would rather have a static final fields with smth like "modifierSuffixes" and just check all of them here. That way if this breaks and a new one is added it is much faster to fix it.